schemas/maas-backend/customers/retrieve/response.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/customers/retrieve/response.json",
+  "description": "MaaS customer retrieve",
+  "type": "object",
+  "properties": {
+    "customer": {
```

Comment:
Why nest in `customer`? Wouldn't it be cleaner to expose `customer` properties directly?

Reply:
@medikoo imo it's a good practice having a wrapper property, as like itinerary and booking API response as we have itinerary and booking as a wrapper property